start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
@rht I still do not understand. If you run this from the terminal, it results in `NameError: name 'isTravis' is not defined`. This would be expected, since isTravis is not declared. The only two solutions I can see are to put it in a try-except clause or to switch the GUI off by default (as suggested by @davidrpugh)